src/SpecStore.ts

```diff
@@ -357,5 +420,6 @@ export default class SpecStore {
       clearTimeout(this.idListsSyncTimer);
       this.idListsSyncTimer = null;
     }
+    // TODO: handle shutdown for config adapter
```

Comment:
Stopping connection to the Redis client requires calling the async function `.quit()`. But the Statsig `shutdown` is not async. Should we make this async?